src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -213,5 +249,23 @@ public void CheckOSPlatform()
             Assert.Equal(0, defaultObj.GetHashCode());
             Assert.Equal(defaultObj.GetHashCode(), conObj.GetHashCode());
         }
+
+        [Fact]
+        public void PlatformNamesAreComparedUsingOrdinalIgnoreCase()
+        {
+            Assert.Equal(OSPlatform.Create("ab"), OSPlatform.Create("ab"));
+            Assert.Equal(OSPlatform.Create("AB"), OSPlatform.Create("AB"));
+            Assert.Equal(OSPlatform.Create("AB"), OSPlatform.Create("ab"));
+            Assert.Equal(OSPlatform.Create("aB"), OSPlatform.Create("Ab"));
+        }
+
+        [Fact]
+        public void OSX_equals_macOS()
+        {
+            Assert.Equal(OSPlatform.OSX, OSPlatform.macOS);
```

Comment:
Suggest adding:

``` csharp
Assert.Equal(OSPlatform.Create("macOS"), OSPlatform.Create("osx"));
```

Reply:
You added similar tests while I was reviewing, but something that hits the case insensitive match on “OSX” would be good.